### PR TITLE
Add Code to return correct units and match PEP8

### PR DESCRIPTION
**Pull request by SConaway**
Sorry about the reformating, that was my editor

### CODE REVIEW

**Comment by Petro31** [comment]
Please do not add this PR.  This should be handled by the client.

**Comment by Petro31** [comment]
Plus looking at the code, you don't even perform the conversion.

**Comment by SConaway** [comment]
Waze actually returns it with the correct units, just this library defaults to km.

**Comment by SConaway** [comment]
If Waze only returned one or the other, I'd agree about client-side handling.

**Comment by kovacsbalu** [comment]
I tested (master branch) with some random USA address:
```
region='US'
From: 855 El Camino Real, Palo Alto, CA 94301 - to: 2615 Broadway, Redwood City, CA 94063
Time 13.47 minutes, distance 8.73 km.
```
https://www.waze.com/en-GB/livemap with same addresses is
```
13 min
5.4 miles
Fastest
El Camino Real Menlo Park
```
As I know 8.73 km == 5.43 miles

File: WazeRouteCalculator/WazeRouteCalculator.py [hunk 1]
```diff
@@ -56,22 +56,31 @@ def __init__(self, start_address, end_address, region='EU', vehicle_type='', log
...
 
-        if self.already_coords(start_address): #See if we have coordinates or address to resolve
+        # See if we have coordinates or address to resolve
+        if self.already_coords(start_address):
             self.start_coords = self.coords_string_parser(start_address)
         else:
             self.start_coords = self.address_to_coords(start_address)
-        self.log.debug('Start coords: (%s, %s)', self.start_coords["lat"], self.start_coords["lon"])
+        self.log.debug('Start coords: (%s, %s)',
+                       self.start_coords["lat"], self.start_coords["lon"])
```
Comment (kovacsbalu) [inline WazeRouteCalculator/WazeRouteCalculator.py]: Please don't reformat 'long' lines.

**Comment by SConaway** [comment]
Weird. I wonder if the server was returning the wrong value for a time.

**Pull request closed without merging**